src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -409,7 +410,7 @@ public void mergeAlignment(final File referenceFasta) {
                         final SAMRecord matePrimary = isRead1 ? r2Primary : r1Primary;
 
                         for (final SAMRecord supp : supplementals) {
-                            final SAMRecord out = clone(sourceRec);
+                            final SAMRecord out = sourceRec.deepCopy();
```

Comment:
The only place in the code that I could find that reverse complements is ::makeReadUnmaped() is that the problem? Perhaps a better solution would be to add a boolean flag to this method that will copy the bases rather than do an in-place reverse complementing of the bases and qualities?